client/src/app/tabs/EngineProfile.js

```diff
@@ -152,6 +154,11 @@ function EditableVersionSection(props) {
   const engineLabel = ENGINE_LABELS[executionPlatform];
   const name = 'engineProfile.version';
 
+  const currentMinorVersion = toSemverMinor(engineProfile.executionPlatformVersion);
+  const minorVersions = engineProfileVersions.map(toSemverMinor);
+
+  const isKnowVersion = isKnownVersion(engineProfileVersions, engineProfile.executionPlatformVersion);
```

Comment:
`isKnownVersion` -> typo :)

Reply:
Fixed in [9090d6d](https://github.com/camunda/camunda-modeler/pull/4414/commits/9090d6de0d1099de887332c28459efa76da4d5a5) :)